models/rwc2018/rooms.txt

```diff
@@ -0,0 +1,27 @@
+kitchen; min1; 6,10; -4,60
```

Comment:
Why is this in a textifle?

Reply:
This is used as input to calc_rooms. Although this the old one from rwc2017. So I will remove it before merging.